pkg/apis/core/validation/validation.go

```diff
@@ -5290,6 +5295,19 @@ func ValidatePodUpdate(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 	return allErrs
 }
 
+func isPodResizeRequestValid(pod core.Pod) bool {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+		return false
+	}
+	for _, c := range pod.Status.ContainerStatuses {
```

Comment:
Add a comment here explaining what this is doing and why (i.e. for version skew, and why it works)

Reply:
Also, leave a TODO to remove this code after GA+3 releases.